docs/gen-ai/gen-ai-metrics.md

```diff
@@ -171,6 +175,106 @@ Instrumentations SHOULD document the list of errors they report.
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+## Generative AI Model Server Metrics
+
+The following metric instruments describe Generative AI model servers'
```

Comment:
can we please render attribute table for each metric? (Or if all attributes are the same, could we render them here just once?)

Reply:
Added